app/helpers/hosts_and_hostgroups_helper.rb

```diff
@@ -60,7 +60,7 @@ def puppet_ca(f, can_override, override)
     select_f f, :puppet_ca_proxy_id, proxies, :id, :name,
              { :include_blank => blank_or_inherit_f(f, :puppet_ca_proxy),
                :disable_button => can_override ? _(INHERIT_TEXT) : nil,
-               :disable_button_enabled => override && !params[:host][:puppet_ca_proxy_id],
+               :disable_button_enabled => override && !(params[:host] && params[:host][:puppet_ca_proxy_id]),
```

Comment:
This looks like a fix for #11255, which already has a PR open - could you remove it?